src/net_processing.cpp

```diff
@@ -2022,6 +2041,61 @@ static bool PrepareBlockFilterRequest(CNode* pfrom, const CChainParams& chain_pa
     return true;
 }
 
+/**
+ * Handle a cfheaders request.
+ *
+ * May disconnect from the peer in the case of a bad request.
+ *
+ * @param[in]   pfrom           The peer that we received the request from
+ * @param[in]   vRecv           The raw message received
+ * @param[in]   chain_params    Chain parameters
+ * @param[in]   connman         Pointer to the connection manager
+ */
+static void ProcessGetCFHeaders(CNode* pfrom, CDataStream& vRecv, const CChainParams& chain_params,
+                                CConnman* connman)
```

Comment:
```suggestion
                                CConnman& connman)
```

Any reason why this is a pointer, but missing the nullptr check?

* Either this should be a proper pointer with nullptr check, or
* it should be a reference

Mixing the two combines the worst of each option.

(Same for pfrom)

Reply:
This usage is the same as all the other `Process*()` functions in net_processing. Rather than introduce a new style for just this function, I think it makes sense to change it everywhere in net_processing in a refactor PR.

Reply:
Generally new code should be style-conformant. See for example `RelayTransaction`, which properly passes connman. Obviously a non-blocking nit, so feel free to close this conversation.